### PR TITLE
adds sqlpad package

### PR DESCRIPTION
**Pull request by mritunjaysharma394**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: sqlpad.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package:
+  name: sqlpad
+  version: 7.1.3
```
Comment (rawlingsj) [inline sqlpad.yaml:3]:
```suggestion
  version: 7.1.3 # when updating check the patch below as it contains dependency version updates which may downgrade if upstream upgrades them
```

File: sqlpad.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - nodejs-18
+      - yarn
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - build-base
+      - yarn
+      - nodejs-18
```
Comment (rawlingsj) [inline sqlpad.yaml:19]: just checking - does this have to be pinned to `nodejs-18`?  Am wondering if `nodejs` would be fine which today is nodejs-20.  Just means we don't have to manage this in the future.

Comment (mritunjaysharma394) [reply sqlpad.yaml:19]: As much I remember, when I tried last, it failed and had to check sqlpad and they supported version 18 that's why I pinned it, let me re-try

Comment (mritunjaysharma394) [reply sqlpad.yaml:19]: Here https://github.com/sqlpad/sqlpad/blob/master/.nvmrc

Comment (mritunjaysharma394) [reply sqlpad.yaml:19]: No worries at all, just done, thanks

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**